src/index.test.ts

```diff
@@ -307,6 +307,8 @@ test('Dependencies range types', async (t) => {
   for (const [name, range] of Object.entries(ourPeerDeps as Record<string, string>)) {
     if (name === 'typescript') {
       t.is(range, '*', 'Peer dependency typescript is `*`')
+    } else if (name === 'eslint-plugin-n') {
```

Comment:
Without this, goes directly to `isSingleCaretRange`, I don't know if there's a better way to do this ¿'?